arlas-core/src/main/java/io/arlas/server/app/Documentation.java

```diff
@@ -30,17 +30,44 @@
     public static final String OPENSEARCH_CSW_OPERATION = "Access to the OpenSearch CSW Description document";
     public static final String PROJECTION_PARAM_INCLUDE = "List the name patterns of the field to be included in the result. Seperate patterns with a comma.";
     public static final String PROJECTION_PARAM_EXCLUDE = "List the name patterns of the field to be excluded in the result. Seperate patterns with a comma.";
-    public static final String SIZE_PARAM_SIZE = "The maximum number of entries or sub-entries to be returned. The default value is 10";
-    public static final String SIZE_PARAM_FROM = "From index to start the search from. Defaults to 0.";
-    public static final String SORT_PARAM_SORT = "- Sort the result on the given fields ascending or descending. " +
+    public static final String PAGE_PARAM_SIZE = "The maximum number of entries or sub-entries to be returned. The default value is 10";
+    public static final String PAGE_PARAM_FROM = "From index to start the search from. Defaults to 0.";
+    public static final String PAGE_PARAM_SORT = "Sorts the resulted hits on the given fields and/or by distance to a given point :" +
             "\n \n" +
-            "- Fields can be provided several times by separating them with a comma. The order matters. " +
+            "> __**Syntax**__: `{field1},{field2},-{field3},geodistance:{lat} {lon},{field4}  ...`." +
             "\n \n" +
-            "- For a descending sort, precede the field with '-'. The sort will be ascending otherwise." +
+            "> **Note 1**: `{field}` can be preceded by **'-'**  for **descending** sort. By default, sort is ascending." +
             "\n \n" +
-            "- For a geodistance sort, specify the point, from which the distances are calculated, as follow : 'geodistance:lat lon'" +
+            "> **Note 2**: The order of fields matters." +
+            "\n \n" +
+            "> **Note 3** ***geodistance sort***: Sorts the hits centroids by distance to the given **{lat} {lon}** (ascending distance sort). It can be specified at most 1 time." +
+            "\n \n" +
+            "> __**Example 1**__: sort=`age,-timestamp`. Resulted hits are sorted by age. For same age hits, they are decreasingly sorted in time." +
+            "\n \n" +
+            "> __**Example 2**__: sort=`age,geodistance:89 179`. Resulted hits are sorted by age. For same age hits, they are sorted by closest distance to the point(89°,179°)" +
+            "\n \n";
+    public static final String PAGE_PARAM_AFTER = "List of values of fields present in sort param that are used to search after. " +
```

Comment:
Shouldn't it be put in a property file?

Reply:
If we decide to output this content in a property file, it may be done in another PR. By the way, since it's not something that can be updated at runtime, it can stay in source files for the moment.